vpr/src/draw/draw.h

```diff
@@ -48,7 +52,23 @@ ezgl::color to_ezgl_color(vtr::Color<float> color);
 ezgl::color to_ezgl_color(t_color color);
 ezgl::color to_ezgl_color(color_types color_enum);
 
-//ezgl functions
 void draw_screen();
 
+// search bar related functions defined in search_bar.cpp
+void search_and_highlight(GtkWidget *widget, ezgl::application *app);
+bool highlight_rr_nodes(int hit_node);
+void auto_zoom_rr_node(int rr_node_id);
+void highlight_blocks(ClusterBlockId clb_index);
```

Comment:
These various highlight_* functions appear to be prototyped here, but defined in search_bar.cpp.

It seems like they might fit more into draw.cpp (since they aren't searchbar specific).
Regardless its always good to line up the prototypes and implementation so the are in the same related files (e.g. draw.h/draw.cpp, or search_bar.h/search_bar.cpp).